package.json

```diff
@@ -81,7 +81,7 @@
     "gulp-trim": "^1.0.2",
     "gulp-uglify": "^3.0.1",
     "husky": "^1.1.1",
-    "lerna": "^3.0.0-rc.0",
+    "lerna": "^3.14.1",
```

Comment:
Upgrading lerna and browser-sync sounds good, but can we pull these into a separate PR?

Reply:
@mwcz sure i'll remove from this PR, not sure why its appearing here. Do you have this captured as an issue or something elsewhere?

Reply:
#90